android/expoview/src/main/java/host/exp/exponent/notifications/presenters/modifiers/ColorModifier.java

```diff
@@ -0,0 +1,15 @@
+package host.exp.exponent.notifications.presenters.modifiers;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.support.v4.app.NotificationCompat;
+
+public class ColorModifier implements NotificationModifierInterface {
```

Comment:
I kind of like this concept of Modifiers. However, it might be a little overengineered in some places.
For example, I believe we should stick with all visual (at least basic) and required staff in one place. It wouldn't be huge, but much easier to refactor/change in the future.